src/ukify/ukify.py

```diff
@@ -64,6 +64,24 @@ def shell_join(cmd):
     return ' '.join(shlex.quote(str(x)) for x in cmd)
 
 
+def check_path_exists(p):
```

Comment:
I think a better name for this would be `check_path_is_accessible` or `check_path_is_readable` or either without the `_is`, so as to avoid confusion with the `pathlib.Path.exists()` method.